modules/phase_field/src/actions/BicrystalBoundingBoxICAction.C

```diff
@@ -72,9 +72,6 @@ BicrystalBoundingBoxICAction::act()
       poly_params.set<Real>("inside") = 0.0;
       poly_params.set<Real>("outside") = 1.0;
     }
-    if (isParamValid("block"))
```

Comment:
if my understanding is correct (and I just checked also) the applyParameters above does the job of adding block